src/pretix/base/models/items.py

```diff
@@ -122,6 +122,16 @@ def __str__(self):
             return _('{category} (Add-On products)').format(category=str(name))
         return str(name)
 
+    def get_category_type_display(self):
+        if self.is_addon:
+            return _('Add-On products')
+        else:
+            return None
+
+    @property
+    def category_type(self):
+        return 'addon' if self.is_addon else 'normal'
+
```

Comment:
I do not see it used elsewhere (yet?), so is it worth the added „complexity“ instead of doing a `{% if c.is_addon %} <span class="font-normal">({% trans "Add-On products" %})</span>{% endif %}` in the (currently only one) template?

Reply:
I did this way, because this PR is somewhat of a "backport" from [the cross-selling PR](https://github.com/pretix/pretix/pull/4185), where I first implemented this and then split it out into this separate PR. Once the cross-selling feature lands, we need a more complex logic which I think justifies the additional code here:
```python

    def __str__(self):
        name = self.internal_name or self.name
        category_type = self.get_category_type_display()
        if category_type:
            return _('{category} ({category_type})').format(category=str(name), category_type=category_type)
        return str(name)

    def get_category_type_display(self):
        if self.is_addon:
            return _('Add-On products')
        elif self.cross_selling_mode:
            return self.get_cross_selling_mode_display()
        else:
            return None

    @property
    def category_type(self):
        return 'addon' if self.is_addon else self.cross_selling_mode or 'normal'

```

Reply:
Makes total sense, thanks for clearing that up!